packages/flutter_driver/test/retry_test.dart

```diff
@@ -51,6 +51,24 @@ main() {
       });
     });
 
+    test('obeys predicates', () {
+      fakeAsync.run((_) {
+        int retryCount = 0;
+
+        expect(
```

Comment:
i don't understand what this is testing. Maybe a short comment explaining it might be good?